test/libsolidity/util/BytesUtils.cpp

```diff
@@ -252,7 +252,35 @@ string BytesUtils::formatBytes(
 		if (*_bytes.begin() & 0x80)
 			os << formatSigned(_bytes);
 		else
-			os << formatUnsigned(_bytes);
+		{
+			std::string hex(formatHex(_bytes));
+			unsigned int value = u256(_bytes).convert_to<unsigned int>();
+			if (value < 10)
+				os << dec;
+			else if (value >= 10 && value <= 0xff) {
+				os << hex;
+			}
+			else
+			{
+				auto entropy = [](std::string const& str) -> double {
+					double result = 0;
+					std::map<char, int> frequencies;
+					for (char c: str)
+						frequencies[c]++;
+					for (std::pair<char, int> p: frequencies)
```

Comment:
```suggestion
					for (auto const& [character, frequency]: frequencies)
```

Reply:
or count?

Reply:
I did this but realised that only the second parameter (e.g. `frequency` or `count`) is used here. I would need to do something like `(void)character;` to get rid of the unused variable warning. I think the original loop is then nicer.

Reply:
Also `std::accumulate()` can be considered.

Reply:
> I would need to do something like (void)character; to get rid of the unused variable warning.

Side remark. Interesting, I just tried, and it seems that in case of structured binding the compilers do not emit the unused variable warning.

Reply:
That's really interesting. A CI build was failing because of this, I don't remember what compiler was used. On my machine I also don't see that unused variable warning.